src/dom-to-image-more.js

```diff
@@ -122,6 +147,34 @@
                 return clone;
             });
         }
+
+        function makeSvgDataUri(node) {
+            let width = options.width || util.width(node);
+            let height = options.height || util.height(node);
+
+            return Promise.resolve(node)
+                .then(function (svg) {
+                    svg.setAttribute('xmlns', 'http://www.w3.org/1999/xhtml');
+                    return new XMLSerializer().serializeToString(svg);
+                })
+                .then(util.escapeXhtml)
+                .then(function (xhtml) {
+                    const foreignObjectSizing =
+                        (util.isDimensionMissing(width)
+                            ? ' width="100%"'
+                            : ` width="${width}"`) +
+                        (util.isDimensionMissing(height)
+                            ? ' height="100%"'
+                            : ` height="${height}"`);
+                    const svgSizing =
+                        (util.isDimensionMissing(width) ? '' : ` width="${width}"`) +
```

Comment:
The `svg` object is now given a size because if we're wrapping a `span` or other inline component without a intrinsic size we need to use the size it would have been rendered at.